assets/js/shortcuts.js

```diff
@@ -81,6 +84,18 @@ function remToPx(rem) {
 }
 
 function setupShortcuts(shortcutDepth = 2) {
+  // Find the shortcut target container in the sidebar
+  const shortcutsTarget = document.getElementById("shortcuts");
+
+  /*
+   * Exit early if shortcuts container doesn't exist
+   * This is important to avoid errors when the theme
+   * is used on pages that don't have the shortcuts sidebar
+   */
+  if (!shortcutsTarget) {
+    return;
+  }
```

Comment:
This check sets up the shortcuts when the page first loads (once), differing from the above one as it prevents attempting to build the shortcuts secyion if there is nowhere to put them.